mac: fix context menus

csett86 opened the pull request:
A mac specific issue in Qt 5.15.0 - at least 5.15.2, see
https://bugreports.qt.io/browse/QTBUG-97559

Closes: #822

csett86 committed:
mac: Fix Context menu in paths in pattern piece tool

A mac specific issue in Qt 5.15.0 - at least 5.15.2, see
https://bugreports.qt.io/browse/QTBUG-97559

Closes: #822

csett86 commented:
This works on mac, still need to test whether this breaks on linux or windows...

DSCaskey commented:
There's more places where QMenu() is used in context menus. Specifically the Internal Paths and Insert Nodes dialogs. I'm assuming Mac doesn't lik the menus not having a parent?

csett86 commented:
The bug seems to be limited to context menus in modal dialogs, and its also fixed in eg 6.2.0 if I read the Qt bug ticket correctly.

csett86 committed:
mac: fix context menu issues in other places

A mac specific issue in Qt 5.15.0 - at least 5.15.2, see
https://bugreports.qt.io/browse/QTBUG-97559

csett86 commented:
But thanks for the hint, now fixed with the same workaround in the other places where there was no parent
